src/Program.cs

```diff
@@ -116,6 +116,9 @@ private T GetXmsCodeGenSetting<T>(CodeModel codeModel, string name)
                 Settings.Instance.CustomSettings.Add("InternalConstructors", GetXmsCodeGenSetting<bool?>(codeModelT, "internalConstructors") ?? await GetValue<bool?>("use-internal-constructors") ?? false);
                 Settings.Instance.CustomSettings.Add("SyncMethods", GetXmsCodeGenSetting<string>(codeModelT, "syncMethods") ?? await GetValue("sync-methods") ?? "essential");
                 Settings.Instance.CustomSettings.Add("UseDateTimeOffset", GetXmsCodeGenSetting<bool?>(codeModelT, "useDateTimeOffset") ?? await GetValue<bool?>("use-datetimeoffset") ?? false);
+                var useLegacyConstructors = await GetValue<bool?>("use-legacy-constructors") ?? false;

```

Comment:
So, we're switching the default behavior to use the new constructors. nice.

Reply:
well, we should think that through again... not sure we wanna change ~60 readme.md's to not break current code gen just because of this